cmd/agent/daemon/daemon.go

```diff
@@ -45,6 +44,8 @@ var (
 
 	mutedNamespaces = pflag.StringArray("mute-namespace", []string{"kube-system", "calico"}, "List of namespaces to ignore tracing events for. To mute multiple namespaces, provide this flag multiple times.")
 
+	disableFileHashEnricher = pflag.Bool("disable-file-hash-enricher", false, "Disables the file has event enricher for exec events")
```

Comment:
I think for opt in features and new features it would be better to have enable with default false instead. Also in all other places inside controller we have enable-something